tests/unit_tests/analysis/test_es_update.py

```diff
@@ -566,6 +584,9 @@ def g(X):
             iens,
         )
 
+    prior_ensemble.unify_parameters()
+    prior_ensemble.unify_responses()
```

Comment:
Would this also work without doing `unify`? So we have tests of not doing that as well.

Reply:
Seems to not work now. Because of this in `get_measured_data`:
```
        for response_type in sorted(self.experiment.observations):
            obs_datasets = self.experiment.observations[response_type]
            # obs_keys_ds = xr.Dataset({"obs_name": observation_keys})
            obs_names_to_check = set(obs_datasets["obs_name"].data).intersection(
                observation_keys
            )
            responses_ds = self.load_responses(
                response_type,
```

Which does expose an "edge case": `load_responses(...)` called with a `response type` as key, it will only work for unified, as there is no `gen_data.nc` file in response folders. For summary it would work, but I'm not sure if we should support this, the "solution" would be to create a combined `gen_data` on the fly, return it without storing it to a file.

Reply:
So there are some calls that require a joined dataset? How is the error message if you do that?

Reply:
The error will be the same as if it doesn't find a response in the realizations folder:
```            
if len(datasets) == 0:
                raise KeyError(
                    f"No response for key {key}, realization: {realizations}"
                ) from None```

Reply:
But doesnt this mean that the fallback to reading a non-unified dataset does not work?